src/components/user_items_page/UserItems.js

```diff
@@ -2,28 +2,40 @@ import React, { useState, useEffect } from 'react';
 import ItemsPage from 'components/all_items_page/ItemsPage';
 import UserItemCard from './UserItemCard';
 import NoContent from 'components/UI/NoContent';
+import Error from 'components/UI/Error';
 
 const UserItems = (props) => {
     const [userItems, setUserItems] = useState(null);
-    const [error, setError] = useState(null);
+    const [errors, setError] = useState(null);
     const { apiHandler, match: { params: { uuid }} } = props;
 
+    const fetchBids = async () => {
+        const { data, errors } = await apiHandler.get(`/api/users/${uuid}/items`);
+        console.log(data)
+        data ? setUserItems(data) : setError(errors);
+    }
+    
     useEffect(() => {
-        const fetchBids = async () => {
-            const { data, error } = await apiHandler.get(`/api/users/${uuid}/items`);
-            data ? setUserItems(data) : setError(error);
-        }
         fetchBids();
     }, [apiHandler, uuid]);
 
-    const handleError = (err) => {
-        return err && `Error: ${err.message}`;
+    const onDelete = async (user_uuid, item_uuid) => {
+        const { status, message } = await apiHandler.delete(`/api/users/${user_uuid}/items/${item_uuid}`)
+        if (status) {
+            fetchBids()
+        } else {
+            if (message.includes("401")) {
```

Comment:
I'd probably prefer that errors are handled through something like specific error codes rather than the status codes that are part of a message. For instance, you can have multiple 400 errors that you'd want to have different error messages for, right? In that case, you'd probably refactor all of these to be different anyway.

Reply:
I haven't needed to render a specific message other than for "unauthorized" errors, so I've kept it like this for now. I was planning on extracting it out later if I see that I was creating custom error messages for different statuses

Reply:
Well, I think this is more of a problem at the API level. If you expect the client to utilize your errors and uniquely identify them, it should provide something like an `errorCode` in the body. Having the client rely on the status code from an api response isn't good design.

Also, another note - having `status` and `message` as the values you pull from the `await` response is a bit confusing. It's hard to tell that `status` only exists on the successful response and `message` only on the error response. It'd be helpful to have an error object of its own returned rather than just a string `message` that's pull from the error object.

Reply:
I took another look at the errors that were coming back from the server, and there IS a "errors" object nested within the response. I've revised the Errors handling in: https://github.com/rebay-8th-light-ebay-clone/rebay/pull/12